cascade/proxima/__init__.py

```diff
@@ -193,7 +209,22 @@ def calculate(
         target_calc: Calculator = self.parameters['target_calc']
         target_calc.calculate(atoms, properties, system_changes)
         self.target_invocations += 1
-        self.results = target_calc.results.copy()
+
+        if self.used_surrogate and blend_with_target:
```

Comment:
Can "blend_with_target" end up being negative?

Reply:
I only add one to it or set it to zero, so it shouln't

Reply:
oh wait...

Reply:
fixed, handleds in tests